docs/src/main/sphinx/security/tls.rst

```diff
@@ -12,17 +12,68 @@ This topic describes how to configure your Trino server to use :ref:`TLS
 All authentication technologies supported by Trino require configuring TLS as
 the foundational layer.
 
-When configured to use TLS, a Trino server responds to client connections using
-TLS 1.2 and TLS 1.3 certificates. The server rejects TLS 1.1, TLS 1.0, and all
-SSL format certificates.
-
 .. important::
 
     This page discusses only how to prepare the Trino server for secure client
     connections from outside of the Trino cluster to its coordinator.
 
 See the :doc:`Glossary </appendix/glossary>` to clarify unfamiliar terms.
 
+.. _tls-version-and-ciphers:
+
+Supported standards
+-------------------
+
+When configured to use TLS, the Trino server responds to client connections
+using TLS 1.2 and TLS 1.3 certificates. The server rejects TLS 1.1, TLS 1.0, and
+all SSL format certificates.
+
+The Trino server does not specify a set of supported ciphers, instead deferring
```

Comment:
This statement and the next paragraph are conflicting. Either Trino can exclude and include ciphers or it cannot.